### PR TITLE
chore: release v0.20.0

### PR DESCRIPTION
**Pull request by niklasad1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,48 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [v0.20.0] - 2023-08-11
+
+Another breaking release were the major changes are:
```
Comment (niklasad1) [inline CHANGELOG.md:9]:
```suggestion
Another breaking release and the major changes are:
```

Comment (lexnv) [inline CHANGELOG.md]:
```suggestion
Another breaking release where the major changes are:
```

**Review by lexnv — APPROVED** [review]
Nice! Useful changes and amazing changelog!

**Review by jsdw — APPROVED** [review]
Woop!

**Pull request merged**